modin/pandas/base.py

```diff
@@ -577,7 +577,9 @@ def apply(
         axis = self._get_axis_number(axis)
         ErrorMessage.non_verified_udf()
         if isinstance(func, str):
-            result = self._query_compiler.apply(func, axis=axis, *args, **kwds)
+            if axis == 1:
```

Comment:
Why is it only relevant if `axis=1`? Please add a comment in the code explaining the edge case.

Reply:
This is done to avoid passing `axis` parameter to the Series, which doesn't support it. Corresponding comment is added.